src/styles/blocks/calendar.scss

```
@@ -0,0 +1,69 @@
+.calendar {
+  max-width: max-width($week, $day-size, $day-shift);
+  display: flex;
+  flex-wrap: wrap;
+
+  &__day {
+    @include square($day-size);
+    box-sizing: border-box;
+    background-color: $main-color;
+    border: 1px solid black;
+    text-align: center;
+    position: relative;
+    transition: color 0.5s, transform 0.5s;
+    @include square-shift($day-shift);
+
+    &:hover {
+      background-color: $secondary-color;
+      cursor: pointer;
+      transform: translateY(-20px);
+    }
+
+    @for $index from 1 through $month-length {
+      &:nth-child(#{$index})::before {
+        content: '#{$index}';
+        position: absolute;
+        font-size: 30px;
+        left: 0;
+        right: 0;
+        top: 50%;
+        transform: translateY(-50%);
```

just center your numbers with flex in `calendar__day`
```suggestion
        font-size: 30px;
```